octo_client/client.py

```diff
@@ -13,6 +13,8 @@
 class OctoClient(object):
     """
     HTTP client for OCTo (Open Connection for Tourism) APIs.
+
+    This supports version: https://docs.octo.travel/docs/octo/r6gduoa5ah5ne-octo-api
```

Comment:
This is a good start but we're still far away from supporting this version. We would need to compare all the schemas of each object and support all the endpoints. So I would wait with the merge until the client will be fully compatible otherwise we will release a version frozen somewhere between the old and the new OCTo.